pymeasure/instruments/eurotest/eurotestHPP120256.py

```diff
@@ -0,0 +1,334 @@
+        :param check_period: voltage output will be measured every check_period (seconds) time
+        :param timeout: time (seconds) give to the voltage output to reach the voltage setting
+        :return: None
+        :raises: Exception if the voltage output can't reach the voltage setting
+        before the timeout completes (seconds)
+        """
+        log.info("Executing the wait_for_voltage_output_set function.")
+
+        ref_time = time.time()
+        future_time = ref_time + timeout
+
+        voltage_output_setting = self.voltage
+        time.sleep(self.COMMAND_DELAY)
+        error = 0.02  # error of +-ten volts (0.01kV) to enter into the voltage stable window
+        voltage_output = self.measure_voltage
+        voltage_output_set = (voltage_output > (voltage_output_setting - error)) and \
+                             (voltage_output < (voltage_output_setting + error))
+
+        while not voltage_output_set:
+            actual_time = time.time()
+            log.debug(f"\tWaiting for voltage output set. "
+                      f"Reading output voltage every {check_period} seconds.\n"
+                      f"\tTimeout: {timeout} seconds. Elapsed time: "
+                      f"{round(actual_time - ref_time, ndigits=1)} seconds.")
+
+            time.sleep(check_period)  # wait for voltage output reaches the voltage output setting
+            voltage_output = self.measure_voltage
+            voltage_output_set = (voltage_output > (voltage_output_setting - error)) and \
+                                 (voltage_output < (voltage_output_setting + error))
+            log.debug("voltage_output_valid_range: "
+                      "[" + str(voltage_output_setting - error) +
+                      ", " + str(voltage_output_setting + error) + "]")
+            log.debug("voltage_output: " + str(voltage_output))
+            if actual_time > future_time:
+                raise Exception("Timeout for wait_for_voltage_output_set function")
+
+        # if you are here is because the voltage_setting
+        # has been reaches at the output of the HV Voltage source
+
+    def ask(self, command):
+        """ Overrides Instrument ask method for including query_delay time on parent call.
+        :param command: Command string to be sent to the instrument.
+        :returns: String returned by the device without read_termination.
+        """
+        return super().ask(command, self.query_delay)
+
+    # Constructor
+    def __init__(self,
+                 adapter,
+                 includeSCPI=False,
+                 query_delay=0.3,
+                 timeout=5000,
+                 **kwargs):
+
+        super().__init__(
+            adapter,
+            "Euro Test High Voltage DC Source model HPP-120-256",
+            write_termination="\n",
+            read_termination="",
+            send_end=True,
+            includeSCPI=includeSCPI,
```

Comment:
```suggestion
            includeSCPI=False,
```
I think includeSCPI can be safely hardcoded to False.